auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -591,8 +590,6 @@ export default defineComponent({
     }
 
     onMounted(async () => {
-      const accountSettings = getAccountFromSession()
-      await orgStore.syncOrganization(accountSettings?.id)
```

Comment:
Let the higher level sync from account settings take care of this or it conflicts with the orgId on the url on refresh.